common.blocks/menu/menu.bemhtml

```diff
@@ -44,8 +46,16 @@ block('menu')(
         });
     }),
     attrs()(function() {
-        var attrs = { role : 'menu' };
+        var ctx = this.ctx,
+            attrs = {
+                role : 'menu',
+                id : ctx.id,
+                'aria-label' : ctx.ariaLabel,
+                'aria-labelledby' : ctx.ariaLabelledBy
```

Comment:
BTW: [MSDN describes][1] it as `ariaLabelledby` not `ariaLabelledBy`

[1]: https://msdn.microsoft.com/en-us/library/ie/cc848875(v=vs.85).aspx